### ISSUE TITLE
Application is too slow after latest update

### ISSUE THREAD

**Issue by bojanz27** [gitextensions/gitextensions#6092]
Now we have nice(-er?) looking UI and 5 sec delay on every git action like waiting on changes to appear on Commit button

**Comment by RussKie** [comment]
Please fill the issue template provided and provide as much information as possible.
We are aware of several performance issue however they can be attributed to various causes.

**Comment by fabvit** [comment]
Try to hide the left panel (the ones that enumerates branches and remotes) to see if that helps.

**Comment by illfated** [comment]
@bojanz27 : Please copy back the issue template into your comment post (or close this and open a new issue ticket with the template completely filled in) if you want your issue to be investigated by the gitextensions team.

**Comment by illfated** [comment]
- Git Extensions 3.00.00.4433
- Build fca7cf228b481ee8c1b779cf7b882ccdfbdcd1bc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3260.0
- DPI 96dpi (no scaling)

Just to throw in my "2 cents" after upgrading from 2.51.05 to release version 3 just a couple of days ago:
I noticed that the left panel is noticeably slower to draw its contents than the rest of the GE UI panels, at least 5 seconds or more.
In 2.51.05, I never turned on the left panel (if that feature was there in the first place).
Other than that, the UI in version 3 definitely looks more rich and friendly than the old one.

After re-reading the comments above, I decided that I personally don't need the left panel, since I only use 1 repository tree with only 1 branch visible ("Show current branch only").
I sometimes create temporary branches for minor pull requests, but I usually delete those branches when they get merged into master.
(I am aware that my environment may not be very productive or common enough to compare with the average user, but it is an observation of sorts.)

**Comment by vbjay** [comment]
Also be aware release 3.0 was a major version release.  MAJOR changes.  We are trying to get a maintenance version out soon.  Working on release generation fixes and testing the release version.

**Comment by gerhardol** [comment]
Look at the Git Command Log (F12)
If you have many submodules, try disabling "update submodule in browse"
If internet access is slow, disable Avatars (Browse-View, Settings)

**Comment by RussKie** [comment]
Duplicate of #5884
